### DISCUSSION TITLE
Didn't sync without notice - resumed when opened manually - normal behavior?

### DISCUSSION THREAD

**Discussion by pheraph** [MobiusSync/MobiusSync#68] [General]
After Moebius had actually been syncing flawlessly in the background the whole time, I noticed today that no sync had occurred for about 2 weeks without me noticing. After I opened the app manually, the sync worked directly again. Is this a limitation of iOS or is there anything I can do to prevent this from happening again?

**Comment by SimonPickup** [comment]
Yes, this is an iOS "feature", it stops background tasks eventually if you don't open the app.
